io-connect-desktop-workspaces/README.md

```diff
@@ -0,0 +1,2 @@
+# Workspaces in React
+This project provides a starter for React based Workspaces applications in io.Connect Desktop and io.Connect Browser.
```

Comment:
Template for creating a custom Workspaces App \[for \*\*io.Connect Desktop**](https://docs.interop.io/desktop/capabilities/windows/workspaces/overview/index.html#extending_workspaces) or \[for \*\*io.Connect Browser**](https://docs.interop.io/browser/capabilities/windows/workspaces/workspaces-app/index.html) projects.

\## Prerequisites

For \*\*io.Connect Desktop** projects, \*\*io.Connect Desktop** 9.0 or newer is required. For \*\*io.Connect Browser** 
 projects, \*\*io.Connect Browser** 3.0 or newer is required.

\## Usage

\- Run \`npm install\` to install all dependencies.
\- Run \`npm run start\` to start the app.
\- For \*\*io.Connect Desktop** projects, copy the \`workspaces.json\` configuration file and replace the existing Workspaces App definition in the \`%LocalAppData%\interop.io\io.Connect Desktop\Desktop\config\apps\` folder. For \*\*io.Connect Browser** projects, see the \[Enabling Workspaces](https://docs.interop.io/browser/capabilities/windows/workspaces/enabling-workspaces/index.html) documentation.
\- Start \*\*io.Connect Desktop** to start using and modifying your Workspaces App.